core/src/main/scala/cromwell/util/JsonFormatting/WdlValueJsonFormatter.scala

```diff
@@ -17,10 +15,12 @@ object WdlValueJsonFormatter extends DefaultJsonProtocol {
       case o: WdlObject => new JsObject(o.value map {case(k, v) => k -> write(v)})
       case a: WdlArray => new JsArray(a.value.map(write).toVector)
       case m: WdlMap => new JsObject(m.value map {case(k,v) => k.valueString -> write(v)})
-      case e: WdlExpression => JsString(e.toWdlString)
       case q: WdlPair => new JsObject(Map("left" -> write(q.left), "right" -> write(q.right)))
       case WdlOptionalValue(_, Some(innerValue)) => write(innerValue)
       case WdlOptionalValue(_, None) => JsNull
+        // handles WdlExpression
+      case v: WdlValue => JsString(v.toWdlString)
```

Comment:
This is a sneaky way of dealing with `WdlExpression`s, this may have the side effect of silently accepting junk that previously would have failed hard with match errors.

Reply:
I'm confused why we suddenly need this now though?

Reply:
`WdlExpression` now lives in `wdl`, but `core` depends only on `wom` not `wdl`.